_component_design/buttons-iconography.md

```diff
@@ -0,0 +1,354 @@
+---
+title: Button Icons
+parent: Inputs and Controls
+layout: component
+category: Components
+usage: >
+  An icon button enables a user to perform an action or initiate a process on a Control Panel page. A button directs the user's attention to important actions.
+preview-image: preview-images/button-icon_thumbnail.svg
+status: stable
+resource: true
+last-modified: 2018-09-28
+helix-ui-css: true
+helix-ui-javascript: false
+---
+
+{% include toc.html %}
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Add an icon to a button when you want to visually represent an action to the user.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Best practices
+
+Consider the following best practices when you construct a button that includes an icon:
+
+- Do not add a icon to a button unless it is immediately recognizable to the user. Icons are often culturally defined. For example, shopping carts don’t exist in all areas of the world.
+- Unless an icon is an established standard, pair the icon with text to convey meaning.
+- Be consistent in your use of iconography in a [button bar]({{site.baseurl}}/component/buttons-bar.html). Don’t mix text and icon buttons with text-only buttons.
+- Except drop-down indicator icons, place the icon to the left of the button text.
+- If there is an established meaning to an icon, don’t use it to mean something else. For example, don’t use a cog for Settings. A cog should only be used for a drop-down menu of actions.
+- Don’t deviate from the established secondary button color or tertiary button color.
```

Comment:
do we need to call out secondary & tertiary? is is sufficient to say established button color?

Reply:
Your suggestion makes sense. Done.